File: AIClient_Python/src/world/Team.py
'''
Created on Dec 20, 2014

@author: samuel
'''
from world.Character import Character
from mathUtils.Vector2 import Vector2

class Team(object):

    def __init__(self, teamId, numberOfCharacter):
        self.teamId = teamId
        self.characters = []
        for index in range(0, numberOfCharacter):
            self.characters.insert(index, Character(index, Vector2(0,0)))
    
    def getFirstCharacter(self):
        return self.characters[0]
    
    def getSecondCharacter(self):
        return self.characters[1]

File: AIClient_JAVA/src/world/Team.java
/* ------------------------------------------------------------------------------
 ** _________   _________      ________    _____      _____  ___________ _________
 ** \_   ___ \ /   _____/     /  _____/   /  _  \    /     \ \_   _____//   _____/
 ** /    \  \/ \_____  \     /   \  ___  /  /_\  \  /  \ /  \ |    __)_ \_____  \ 
 ** \     \____/        \    \    \_\  \/    |    \/    Y    \|        \/        \
 **  \______  /_______  /     \______  /\____|__  /\____|__  /_______  /_______  /
 **         \/        \/             \/         \/         \/        \/        \/ 
 **
 ** Team.java
 ** A team
 **
 ** Author: Samuel-Ricardo Carriere
 ** ------------------------------------------------------------------------------*/

package world;

import java.util.ArrayList;
import java.util.List;

public class Team {
	private int id;
	private List<Character> characters = new ArrayList<Character>();

	public Team(int id, int numberOfCharacter) {
		this.id = id;
		for (int index = 0; index < numberOfCharacter; ++index) {
			characters.add(new Character(index));
		}
	}

	public int getId() {
		return id;
	}

	public Character getCharacter(int id) {
		return characters.get(id);
	}

	public List<Character> getCharacters() {
		return characters;
	}
}
